tests/fideslang/test_models.py

```diff
@@ -152,7 +152,7 @@ def test_system_valid(self) -> None:
         ]
         assert system.meta == {"some": "meta stuff"}
         assert system.organization_fides_key == "1"
-        assert system.cookies == [Cookies(name="test_cookie", path=None, domain=None)]
+        assert system.cookies == None
```

Comment:
solid! 👍 may just make a comment here indicating _why_ we're asserting this is `None`

Reply:
added here https://github.com/ethyca/fideslang/pull/27/commits/956b6ec6fefa4fd66e686878317c94fa25b55e15#diff-a50302bafd0d0137a014c5b9a7bfb97481729ab826693473a24b2796c04afd4cR157